doc/dependency_decisions.yml

```diff
@@ -89,3 +89,9 @@
     :license_links: https://github.com/yinzara/protoc-gen-js/blob/master/LICENSE
     :versions: []
     :when: 2023-05-12 14:28:29.173104997 Z
+- - :permit
+  - ISC
+  - :who:
+    :why:
+    :versions: []
+    :when: 2023-10-18 20:31:08.255446206 Z
```

Comment:
[permitted in another repo](https://github.com/viamrobotics/goutils/blob/429830ca0acd683d214b1ed926e64569e1a507ac/doc/dependency_decisions.yml#L27-L31) - similar to MIT / simplified BSD